pkg/utils/xml2text.go

```diff
@@ -32,9 +41,14 @@ func xmlToHtml(in string) string {
 			// ignore errors and try to format as much as possible
 			continue
 		}
+
 		switch tok := t.(type) {
 		case xml.StartElement:
-			builder.WriteString(formatElement(tok.Name, "<"))
+			if preRender && tok.Name.Local == "sub" && len(tok.Attr) > 1 {
```

Comment:
So can the subelement only ever refer to a value or should the substitution be only done if `tok.Attr[0]` has a variable prefix?